### PR TITLE
feat: updated version with shorter code

### PR DESCRIPTION
**Pull request by carevick**

### CODE REVIEW

File: src/js/circuitBreaker.js [hunk 1]
```diff
@@ -1,9 +1,68 @@
...
+        });
+        
+        const img = document.createElement("img");//Represents image for main button
+        img.src = 'img/1.jpg', img.alt= 'Fotografija osigurača';
+        
+        const dialog = document.createElement("dialog");//Represents dialog that contains breaker's label and dialog's close button
+               
+          const dialogContentBox = document.createElement("p");//Represents paragraph for breaker's label
+        
+            const breakerLabel = [];
```
Comment (rdeak) [inline src/js/circuitBreaker.js]: please extract from function

File: src/js/circuitBreaker.js [hunk 2]
```diff
@@ -1,9 +1,68 @@
-function showModal(dialogId){
-    const dialog = document.getElementById(dialogId)
-    dialog.showModal();
-};
+const createElements = ()=> {
```
Comment (rdeak) [inline src/js/circuitBreaker.js]: please parametrize like `createElements = (data) => {`

File: src/js/circuitBreaker.js [hunk 3]
```diff
@@ -1,9 +1,68 @@
...
+      dialog.appendChild(closeButton);//appending <button> element in <dialog> element
+      openDialogButton.appendChild(img);//appending <image> element in main <button> element
+      circuitBreaker.appendChild(openDialogButton);//appending <button> element in <div> element
+      circuitBreaker.appendChild(dialog);//appending <dialog> element in <div> element
+      container.appendChild(circuitBreaker);//appending <div> elements in HTML's (DOM) <div> element
+      
+    }
+  }
+    
+  window.addEventListener('load', createElements());
```
Comment (rdeak) [inline src/js/circuitBreaker.js]: if we parametrize the function then this line can changed to
`window.addEventListener('load', createElements(data));`

**Review by rdeak — COMMENTED** [review]
LGTM, though left some comments how it can be batter

**Review by carevick — COMMENTED** [review]
Style update is good

File: src/js/circuitBreaker.js (hunk 3 above)
Comment (carevick) [reply src/js/circuitBreaker.js]: with line:
`window.addEventListener('load', createElements(data));` 

-> throws: 
`Uncaught ReferenceError: data is not defined`

=> instead changed to:
`window.addEventListener('load', createElements(breakerLabels));`

**Review by rdeak — APPROVED** [review]
:1st_place_medal:

**Pull request merged**